packages/backend-for-frontend/src/routers/supportRouter.ts

```diff
@@ -34,14 +34,11 @@ const supportRouter = (
   );
 
   supportRouter.post("/session/saml2/tokens", async (req, res) => {
-    const { tenantId, saml2 } = req.params;
```

Comment:
This was the bug. We were extracting from params and not from the body. I have no idea why TS was not complaining here or below 🤔